From a21d86b40dc3bb89eca3b62b396845b423065081 Mon Sep 17 00:00:00 2001
From: Gioele Barabucci <gioele@svario.it>
Date: Mon, 14 Mar 2022 18:22:57 +0100
Subject: [PATCH 2/2] debian/control: Remove Pre-Depends on awk

The script `current-md5sums-obsolete` (now removed) was the only
thing that required `awk`.
---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 960c5d4..f761926 100644
--- a/debian/control
+++ b/debian/control
@@ -8,7 +8,6 @@ Build-Depends: debhelper-compat (= 13)
 Package: base-files
 Provides: base
 Architecture: any
-Pre-Depends: awk
 Depends: ${misc:Depends}
 Essential: yes
 Priority: required
-- 
2.34.1

